python/lsst/meas/extensions/photometryKron/kronLib.i

```diff
@@ -37,15 +37,39 @@ Interface to Kron magnitudes
 #include "lsst/afw/detection/Footprint.h"
 #include "lsst/afw/image/Utils.h"
 #include "lsst/afw/geom/Point.h"
-#include "lsst/meas/algorithms.h"
-#include "lsst/meas/extensions/photometryKron.h"
+#include "lsst/afw/math.h"
+#include "lsst/afw/image.h"
+#include "lsst/afw/detection.h"
+#include "lsst/afw/table.h"
+#include "lsst/meas/base.h"
+#define PY_ARRAY_UNIQUE_SYMBOL LSST_MEAS_BASE_NUMPY_ARRAY_API
```

Comment:
I'm not exactly sure why this line is here at all.  I copied it from
another .i file, and thought it was needed.  Please explain what it does.

I am also unclear about the problem with product.  It did not seem to work
when I reenabled testForced in Kron.py.  I worked around it, meaning to ask
you if it worked in previous versions of this tests.

On Wed, Feb 18, 2015 at 5:35 PM, Jim Bosch <notifications@github.com> wrote:

> In python/lsst/meas/extensions/photometryKron/kronLib.i
> <https://github.com/lsst/meas_extensions_photometryKron/pull/1#discussion_r24959794>
> :
>
> > @@ -37,15 +37,39 @@ Interface to Kron magnitudes
> >  #include "lsst/afw/detection/Footprint.h"
> >  #include "lsst/afw/image/Utils.h"
> >  #include "lsst/afw/geom/Point.h"
> > -#include "lsst/meas/algorithms.h"
> > -#include "lsst/meas/extensions/photometryKron.h"
> > +#include "lsst/afw/math.h"
> > +#include "lsst/afw/image.h"
> > +#include "lsst/afw/detection.h"
> > +#include "lsst/afw/table.h"
> > +#include "lsst/meas/base.h"
> > +#define PY_ARRAY_UNIQUE_SYMBOL LSST_MEAS_BASE_NUMPY_ARRAY_API
>
> Yes, these need to be unique across all extension models. @jdswinbank
> <https://github.com/jdswinbank>'s suggestion is correct. (And by the way,
> to get my attention here on GitHub, use @TallJimbo
> <https://github.com/TallJimbo>, but I was already watching this PR
> anyway).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lsst/meas_extensions_photometryKron/pull/1/files#r24959794>
> .
>